openfl-tutorials/interactive_api/Tensorflow_CIFAR_tfdata/director/start_director_with_tls.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Since the README describes running the experiment without TLS, we can get rid of this file.

Reply:
Noted.